src/components/Book.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './book.css';
+
+const Book = (props) => {
+  const {
+    title, category, pages, author, completed, chapter,
+  } = props;
+
```

Comment:
**Changes Required**
- [ ] Let us add a function to remove a book from the store on the `remove button`. Currently remove book is not implemented.